packages/babel-helpers/src/helpers/possibleConstructorReturn.ts

```diff
@@ -1,8 +1,10 @@
 /* @minVersion 7.0.0-beta.0 */
 
-import assertThisInitialized from "assertThisInitialized";
+import assertThisInitialized from "./assertThisInitialized.ts";
 
-export default function _possibleConstructorReturn(self, call) {
+export default function _possibleConstructorReturn<
+  T extends object | undefined,
+>(self: T, call: T | ((...args: any[]) => T) | undefined) {
```

Comment:
While we are here, let's rename `call` to `value`. Calling it `call` makes absolutely no sense 😅 

It's the value passed to `return` in a derived class constructor:
```js
class A extends B {
  constructor() {
    return 1;
  }
}
```

"call" is `1` in this case.

(and probably it's type should just be `any` or `unknown`)

Reply:
ok!!
that makes much sense now!